src/commands/certs/rm.js

```diff
@@ -44,7 +45,16 @@ async function rm(ctx, opts, args, output) {
   }
 
   const id = args[0];
-  const certs = await getCertsToDelete(output, now, id);
+  let certs;
+  try {
```

Comment:
Why do we have to put this in a try/catch? The pattern we follow everywhere is to make the CRUD functions to return known errors so we pattern match and return to exit gracefully rather than running `process.exit(1)`.

That should not provide 1 as exit IMO